index.js

```diff
@@ -102,6 +115,54 @@ function fastifyPostgres (fastify, options, next) {
     }
   }
 
+  fastify.addHook('onRoute', routeOptions => {
+    const useTransaction = routeOptions.useTransaction || (routeOptions.options && routeOptions.options.useTransaction)
```

Comment:
are we allowing two different ways to require a transaction? why?

Reply:
I think this is a misunderstanding on my part about how Fastify works.

When I was using the following syntax I was using an options key

```
method: 'GET',
options: { useTransaction },
handler ...
```

But I think I could put the key directly into that object in which would negate the need for the extra conditions

I will change